src/main/scala/cromwell/engine/package.scala

```diff
@@ -19,7 +19,7 @@ package object engine {
   type WorkflowId = UUID
 
   sealed trait WorkflowState {
-    def isTerminal: Boolean
+    val isTerminal: Boolean
```

Comment:
Having the trait define things as `def` and the implementors going with `val`s is a good thing IMO as it allows for an implementor to make it whatever they want